app/controllers/exchange_rates_controller.rb

```diff
@@ -9,6 +9,8 @@ def show
     @monthly_exchange_rate = ExchangeRates::MonthlyExchangeRate.find(
       "#{year}-#{month}",
     )
+
+    render :show_404, status: :not_found unless @monthly_exchange_rate
```

Comment:
Does this work with your backend change?

So as I understand it, this will:

1. Fetch a response from the backend
2. Try and parse the json api structured json
3. Try and instantiate a monthly exchange rate `new parse_jsonapi(response)`
4. Fail with some unhandled exception
5. Propagate and not render this page

https://github.com/trade-tariff/trade-tariff-frontend/blob/main/lib/api_entity.rb#L234-L238

Reply:
I wonder if we need to extend the ApiEntity implementation to handle 404 responses in the `find` method? I guess we could have it raise some generic `ApiEntity::NotFound` exception

Reply:
sorry this is the wrong PR 1 sec